### ISSUE TITLE
Tests Skipped [ERROR] Unable to connect to plugin

### ISSUE THREAD

**Issue by henkbl** [getgauge/gauge#228]
When running gauge tests on Redhat Linux all tests are skipped and errors are produced. These same tests work on Windows and OSX . Also to note is on RedHat Linux I am able to run the gauge hello world tests found in the getting started guide with success.

The setup I have is:

1. Java runner with the html and xml plugin.

2. OS RedHat 6.5
[vagrant@localhost ~]$ uname -a
Linux localhost.localdomain 2.6.32-573.3.1.el6.x86_64 #1 SMP Mon Aug 10 09:44:54 EDT 2015 x86_64 x86_64 x86_64 GNU/Linux

3. vagrant@localhost ~]$ gauge --version
Gauge version: 0.1.8

Plugins
-------
html-report (1.0.4)
java (0.2.2)
xml-report (0.0.2)

4. For some reason I can't attach a file without getting a error of "Something went really wrong, and we can’t process that file. Try again". So I pasted the output below. The WARN messages show up on the Windows and OSX executions of the same test suite and run with success.

[INFO] Note: Some input files use or override a deprecated API.

[INFO] Note: Recompile with -Xlint:deprecation for details.

[INFO] Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.

[INFO] 1    [main] WARN  org.reflections.Reflections  - could not create Dir using com.thoughtworks.gauge.scan.ClasspathScanner$1@3ff96afb from url file:/home/vagrant/.gauge/plugins/java/0.2.2/*. skipping.
java.lang.NullPointerException
	at com.thoughtworks.gauge.scan.ClasspathScanner$1.createDir(ClasspathScanner.java:53)
	at org.reflections.vfs.Vfs.fromURL(Vfs.java:99)
	at org.reflections.vfs.Vfs.fromURL(Vfs.java:91)
	at org.reflections.Reflections.scan(Reflections.java:237)
	at org.reflections.Reflections.scan(Reflections.java:204)
	at org.reflections.Reflections.<init>(Reflections.java:129)
	at com.thoughtworks.gauge.scan.ClasspathScanner.createReflections(ClasspathScanner.java:60)
	at com.thoughtworks.gauge.scan.ClasspathScanner.scan(ClasspathScanner.java:37)
	at com.thoughtworks.gauge.GaugeRuntime.main(GaugeRuntime.java:36)

[INFO] 4    [main] WARN  org.reflections.Reflections  - could not create Dir using directory from url file:/home/vagrant/.gauge/plugins/java/0.2.2/*. skipping.
java.lang.NullPointerException
	at org.reflections.vfs.Vfs$DefaultUrlTypes$3.matches(Vfs.java:239)
	at org.reflections.vfs.Vfs.fromURL(Vfs.java:98)
	at org.reflections.vfs.Vfs.fromURL(Vfs.java:91)
	at org.reflections.Reflections.scan(Reflections.java:237)
	at org.reflections.Reflections.scan(Reflections.java:204)
	at org.reflections.Reflections.<init>(Reflections.java:129)
	at com.thoughtworks.gauge.scan.ClasspathScanner.createReflections(ClasspathScanner.java:60)
	at com.thoughtworks.gauge.scan.ClasspathScanner.scan(ClasspathScanner.java:37)
	at com.thoughtworks.gauge.GaugeRuntime.main(GaugeRuntime.java:36)

[INFO] 5    [main] WARN  org.reflections.Reflections  - could not create Vfs.Dir from url. ignoring the exception and continuing
org.reflections.ReflectionsException: could not create Vfs.Dir from url, no matching UrlType was found [file:/home/vagrant/.gauge/plugins/java/0.2.2/*]
either use fromURL(final URL url, final List<UrlType> urlTypes) or use the static setDefaultURLTypes(final List<UrlType> urlTypes) or addDefaultURLTypes(UrlType urlType) with your specialized UrlType.
	at org.reflections.vfs.Vfs.fromURL(Vfs.java:109)
	at org.reflections.vfs.Vfs.fromURL(Vfs.java:91)
	at org.reflections.Reflections.scan(Reflections.java:237)
	at org.reflections.Reflections.scan(Reflections.java:204)
	at org.reflections.Reflections.<init>(Reflections.java:129)
	at com.thoughtworks.gauge.scan.ClasspathScanner.createReflections(ClasspathScanner.java:60)
	at com.thoughtworks.gauge.scan.ClasspathScanner.scan(ClasspathScanner.java:37)
	at com.thoughtworks.gauge.GaugeRuntime.main(GaugeRuntime.java:36)

[INFO] java.lang.IncompatibleClassChangeError: class org.springframework.core.LocalVariableTableParameterNameDiscoverer$ParameterNameDiscoveringVisitor has interface org.springframework.asm.ClassVisitor as super class
[INFO]

[INFO] 	at java.lang.ClassLoader.defineClass1(Native Method)
[INFO]

[INFO] 	at java.lang.ClassLoader.defineClass(ClassLoader.java:800)
[INFO]

[INFO] 	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
[INFO]

[INFO] 	at java.net.URLClassLoader.defineClass(URLClassLoader.java:449)
[INFO]

[INFO] 	at java.net.URLClassLoader.access$100(URLClassLoader.java:71)
[INFO]

[INFO] 	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
[INFO]

[INFO] 	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
[INFO]

[INFO] 	at java.security.AccessController.doPrivileged(Native Method)
[INFO]

[INFO] 	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
[INFO]

[INFO] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
[INFO]

[INFO] 	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
[INFO]

[INFO] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
[INFO]

[INFO] 	at org.springframework.core.LocalVariableTableParameterNameDiscoverer.inspectClass(LocalVariableTableParameterNameDiscoverer.java:114)
[INFO]

[INFO] 	at org.springframework.core.LocalVariableTableParameterNameDiscoverer.getParameterNames(LocalVariableTableParameterNameDiscoverer.java:87)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:193)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1035)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:939)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:485)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:456)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:294)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:225)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:291)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:193)
[INFO]

[INFO] 	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1093)
[INFO]

[INFO] 	at com.thoughtworks.twist.gauge.migration.GaugeSpringInitializer.bean(GaugeSpringInitializer.java:92)
[INFO]

[INFO] 	at com.thoughtworks.twist.gauge.migration.GaugeSpringInitializer$1.initialize(GaugeSpringInitializer.java:29)
[INFO]

[INFO] 	at com.thoughtworks.gauge.ClassInstanceManager.get(ClassInstanceManager.java:32)
[INFO]

[INFO] 	at com.thoughtworks.gauge.screenshot.ScreenshotFactory.takeScreenshot(ScreenshotFactory.java:48)
[INFO]

[INFO] 	at com.thoughtworks.gauge.screenshot.ScreenshotFactory.getScreenshotBytes(ScreenshotFactory.java:40)
[INFO]

[INFO] 	at com.thoughtworks.gauge.execution.MethodExecutor.createFailureExecResult(MethodExecutor.java:42)
[INFO]

[INFO] 	at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:36)
[INFO]

[INFO] 	at com.thoughtworks.gauge.execution.HooksExecutor$TaggedHookExecutor.executeHook(HooksExecutor.java:99)
[INFO]

[INFO] 	at com.thoughtworks.gauge.execution.HooksExecutor$TaggedHookExecutor.execute(HooksExecutor.java:85)
[INFO]

[INFO] 	at com.thoughtworks.gauge.execution.HooksExecutor.execute(HooksExecutor.java:42)
[INFO]

[INFO] 	at com.thoughtworks.gauge.processor.MethodExecutionMessageProcessor.executeHooks(MethodExecutionMessageProcessor.java:53)
	at com.thoughtworks.gauge.processor.SuiteExecutionStartingProcessor.process(SuiteExecutionStartingProcessor.java:26)
[INFO]
	at com.thoughtworks.gauge.connection.MessageDispatcher.dispatchMessages(MessageDispatcher.java:72)

[INFO] 	at com.thoughtworks.gauge.GaugeRuntime.main(GaugeRuntime.java:37)

[INFO] java.lang.IncompatibleClassChangeError: class org.springframework.core.LocalVariableTableParameterNameDiscoverer$ParameterNameDiscoveringVisitor has interface org.springframework.asm.ClassVisitor as super class
[INFO]

[ERROR] Unable to connect to plugin Xml Report 0.0.2. proto: required field "SuiteExecutionResult.SuiteResult.PreHookFailure.StackTrace" not set

[ERROR] Unable to connect to plugin Html Report 1.0.4. proto: required field "SuiteExecutionResult.SuiteResult.PreHookFailure.StackTrace" not set

[INFO] Specifications: 	0 executed, 0 passed, 0 failed, 749 skipped
[INFO] Scenarios: 	0 executed, 0 passed, 0 failed
[INFO] Total time taken: 488ms

Thanks
Bob

**Comment by mahendrakariya** [comment]
Thanks for reporting this issue. So far we have tested Gauge on other distros of Linux like Ubuntu. Now, we will test it with Redhat as well.

**Comment by mahendrakariya** [comment]
@henkbl Are you seeing this issue on command line or on IDE?
Also, can you try deleting the `out`, `bin`, and `gauge_bin` folders and try running `gauge specs` again? If there are `.class` files in any other folders, please delete those as well.

Could you also let us know what spring version you are using? 

I'm asking because the error that you have posted here is a known issue with Spring core 3.2.2 + spring security 3.1.4. You can find more details about the issue on [this stackoverflow question](http://stackoverflow.com/questions/16801639/spring-core-3-2-2-spring-security-3-1-4-java-lang-incompatibleclasschangeerro). I just want to make sure Gauge is doing nothing to cause this.

**Comment by henkbl** [comment]
@mahendrakariya 
**Are you seeing this issue on command line or on IDE?** Command line
**Also, can you try deleting the out, bin, and gauge_bin folders and try running gauge specs again? If there are .class files in any other folders, please delete those as well.**  I didn't find an out or bin folder. I did delete the gauge_bin folder and any .class files.
**Could you also let us know what spring version you are using?** Does this tell you the version? spring-core-4.0.7.RELEASE.jar

I was out on vacation for a bit. I re-installed gauge and the plugins on a fresh VM and here are the versions I tested with today.
Gauge version: 0.2.1

Plugins
-------
html-report (1.1.0)
java (0.3.1)
xml-report (0.1.1)

Here is the output when running gauge specs with these versions:

[INFO] Note: Some input files use or override a deprecated API.

[INFO] Note: Recompile with -Xlint:deprecation for details.

[INFO] Note: Some input files use unchecked or unsafe operations.

[INFO] Note: Recompile with -Xlint:unchecked for details.

[INFO] 0    [main] WARN  org.reflections.Reflections  - could not create Dir using com.thoughtworks.gauge.scan.ClasspathScanner$1@7b6393cd from url file:/home/vagrant/.gauge/plugins/java/0.3.1/*. skipping.
java.lang.NullPointerException
	at com.thoughtworks.gauge.scan.ClasspathScanner$1.createDir(ClasspathScanner.java:53)
	at org.reflections.vfs.Vfs.fromURL(Vfs.java:99)
	at org.reflections.vfs.Vfs.fromURL(Vfs.java:91)
	at org.reflections.Reflections.scan(Reflections.java:237)
	at org.reflections.Reflections.scan(Reflections.java:204)
	at org.reflections.Reflections.<init>(Reflections.java:129)
	at com.thoughtworks.gauge.scan.ClasspathScanner.createReflections(ClasspathScanner.java:60)
	at com.thoughtworks.gauge.scan.ClasspathScanner.scan(ClasspathScanner.java:37)
	at com.thoughtworks.gauge.GaugeRuntime.main(GaugeRuntime.java:36)

[INFO] 2    [main] WARN  org.reflections.Reflections  - could not create Dir using directory from url file:/home/vagrant/.gauge/plugins/java/0.3.1/*. skipping.
java.lang.NullPointerException
	at org.reflections.vfs.Vfs$DefaultUrlTypes$3.matches(Vfs.java:239)
	at org.reflections.vfs.Vfs.fromURL(Vfs.java:98)
	at org.reflections.vfs.Vfs.fromURL(Vfs.java:91)
	at org.reflections.Reflections.scan(Reflections.java:237)
	at org.reflections.Reflections.scan(Reflections.java:204)
	at org.reflections.Reflections.<init>(Reflections.java:129)
	at com.thoughtworks.gauge.scan.ClasspathScanner.createReflections(ClasspathScanner.java:60)
	at com.thoughtworks.gauge.scan.ClasspathScanner.scan(ClasspathScanner.java:37)
	at com.thoughtworks.gauge.GaugeRuntime.main(GaugeRuntime.java:36)

[INFO] 10   [main] WARN  org.reflections.Reflections  - could not create Vfs.Dir from url. ignoring the exception and continuing
org.reflections.ReflectionsException: could not create Vfs.Dir from url, no matching UrlType was found [file:/home/vagrant/.gauge/plugins/java/0.3.1/*]
either use fromURL(final URL url, final List<UrlType> urlTypes) or use the static setDefaultURLTypes(final List<UrlType> urlTypes) or addDefaultURLTypes(UrlType urlType) with your specialized UrlType.
	at org.reflections.vfs.Vfs.fromURL(Vfs.java:109)
	at org.reflections.vfs.Vfs.fromURL(Vfs.java:91)
	at org.reflections.Reflections.scan(Reflections.java:237)
	at org.reflections.Reflections.scan(Reflections.java:204)
	at org.reflections.Reflections.<init>(Reflections.java:129)
	at com.thoughtworks.gauge.scan.ClasspathScanner.createReflections(ClasspathScanner.java:60)
	at com.thoughtworks.gauge.scan.ClasspathScanner.scan(ClasspathScanner.java:37)
	at com.thoughtworks.gauge.GaugeRuntime.main(GaugeRuntime.java:36)

[INFO] java.lang.IncompatibleClassChangeError: class org.springframework.core.LocalVariableTableParameterNameDiscoverer$ParameterNameDiscoveringVisitor has interface org.springframework.asm.ClassVisitor as super class
[INFO]

[INFO] 	at java.lang.ClassLoader.defineClass1(Native Method)
[INFO]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:800)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:449)

[INFO] 	at java.net.URLClassLoader.access$100(URLClassLoader.java:71)

[INFO] 	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
[INFO]

[INFO] 	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
[INFO]

[INFO] 	at java.security.AccessController.doPrivileged(Native Method)
[INFO]

[INFO] 	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
[INFO]

[INFO] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
[INFO]

[INFO] 	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
[INFO]

[INFO] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
[INFO]

[INFO] 	at org.springframework.core.LocalVariableTableParameterNameDiscoverer.inspectClass(LocalVariableTableParameterNameDiscoverer.java:114)
[INFO]

[INFO] 	at org.springframework.core.LocalVariableTableParameterNameDiscoverer.getParameterNames(LocalVariableTableParameterNameDiscoverer.java:87)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:193)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1035)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:939)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:485)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:456)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:294)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:225)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:291)
[INFO]

[INFO] 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:193)
[INFO]

[INFO] 	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1093)
[INFO]

[INFO] 	at com.thoughtworks.twist.gauge.migration.GaugeSpringInitializer.bean(GaugeSpringInitializer.java:92)
[INFO]

[INFO] 	at com.thoughtworks.twist.gauge.migration.GaugeSpringInitializer$1.initialize(GaugeSpringInitializer.java:29)
[INFO]

[INFO] 	at com.thoughtworks.gauge.ClassInstanceManager.get(ClassInstanceManager.java:32)
[INFO]

[INFO] 	at com.thoughtworks.gauge.screenshot.ScreenshotFactory.takeScreenshot(ScreenshotFactory.java:48)
[INFO]

[INFO] 	at com.thoughtworks.gauge.screenshot.ScreenshotFactory.getScreenshotBytes(ScreenshotFactory.java:40)
[INFO]

[INFO] 	at com.thoughtworks.gauge.execution.MethodExecutor.createFailureExecResult(MethodExecutor.java:42)
[INFO]

[INFO] 	at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:36)
[INFO]

[INFO] 	at com.thoughtworks.gauge.execution.HooksExecutor$TaggedHookExecutor.executeHook(HooksExecutor.java:99)
[INFO]

[INFO] 	at com.thoughtworks.gauge.execution.HooksExecutor$TaggedHookExecutor.execute(HooksExecutor.java:85)
[INFO]

[INFO] 	at com.thoughtworks.gauge.execution.HooksExecutor.execute(HooksExecutor.java:42)
[INFO]

[INFO] 	at com.thoughtworks.gauge.processor.MethodExecutionMessageProcessor.executeHooks(MethodExecutionMessageProcessor.java:53)
[INFO]

[INFO] 	at com.thoughtworks.gauge.processor.SuiteExecutionStartingProcessor.process(SuiteExecutionStartingProcessor.java:26)
[INFO]

[INFO] 	at com.thoughtworks.gauge.connection.MessageDispatcher.dispatchMessages(MessageDispatcher.java:72)
[INFO]

[INFO] 	at com.thoughtworks.gauge.GaugeRuntime.main(GaugeRuntime.java:37)
[INFO]

[INFO] java.lang.IncompatibleClassChangeError: class org.springframework.core.LocalVariableTableParameterNameDiscoverer$ParameterNameDiscoveringVisitor has interface org.springframework.asm.ClassVisitor as super class
[INFO]

[ERROR] Error Message: Connection closed [127.0.0.1:41486] cause: EOF
[ERROR] Stacktrace:
[ERROR] Error Message: use of closed network connection
[ERROR] Stacktrace:
[ERROR] Unable to connect to plugin Html Report 1.1.0. proto: required field "SuiteExecutionResult.SuiteResult.PreHookFailure.StackTrace" not set

[ERROR] Unable to connect to plugin Xml Report 0.1.1. proto: required field "SuiteExecutionResult.SuiteResult.PreHookFailure.StackTrace" not set

[INFO] Specifications: 	0 executed, 0 passed, 0 failed, 0 skipped
[INFO] Scenarios: 	0 executed, 0 passed, 0 failed, 0 skipped
[INFO] Total time taken: 448ms

Thanks again,
Bob

**Comment by mahendrakariya** [comment]
Hi Bob,

Do you have spring-asm-(version).jar as a dependency? If yes, could you please remove it and try again?

**Comment by henkbl** [comment]
Hello Mahendra,

I found this and deleted it, org.springframework.asm_3.1.0.RELEASE.jar.  I was now able to run my tests with success! woohoo 

Are the steps you had me take manually to get to this point the only way to run gauge on redhat at this time? Let me know if I can provide any more information to help.

Thanks again!
Bob

**Comment by mahendrakariya** [comment]
@henkbl When you raised this issue, we initially thought it could be because of outdated class files. That's the reason we asked you delete the folders `bin` (class file folder used by eclipse), `out` (class file folder used by intellij), and `gauge_bin` (class file folder used by gauge command line). But since that didn't solve the issue, we investigated further.

We found out that it's a spring issue and has nothing to do with the operating system or gauge. In spring 3.2, `spring_asm` and `spring_core` jars have been merged into one. So there is no need to have a separate dependency for `spring_asm`.

For more details, you can see the [Spring 3.2 release announcement](http://spring.io/blog/2012/09/12/spring-framework-3-2-m2-released). They have done these changes in commit [c16f18a5f](https://github.com/spring-projects/spring-framework/commit/c16f18a5fd659f34d6290b5f2cffe54534b7592a).

In a nutshell, you don't have to repeat the manual steps again. It should just work irrespective of the OS you are using.

I will close this issue since it's resolved. Feel free to reopen it if you face it again.

**Comment by haydonduan** [comment]
this case is appear in ruby

Unable to connect to plugin Html Report 2.1.1. proto: required field "SuiteExecutionResult.SuiteResult.PreHookFailure.StackTrace" not set

**Comment by apoorvam** [comment]
@haydonduan Can you please tell which version of Gauge, Ruby plugin and HTML report plugin you are using?

You can run `gauge -v` to find these information.

**Comment by kashishm** [comment]
This happens when language runner dies because of some exception caused by user code and Gauge doesn't get any valid response from the runner.
I could reproduce the exception by adding the following code in my ruby implementation file.

``` ruby
before_suite do
    Kernel.exit!(1)
end
```
**stacktrace** 
```
Ruby runner Failed. Reason: exit status 1
Unable to connect to plugin Html Report 2.2.0. proto: required field "SuiteExecutionResult.SuiteResult.PreHookFailure.StackTrace" not set


  Error Message: Connection closed [[::1]:56038] cause: EOF
  Stacktrace:


  Error Message: write tcp [::1]:56035->[::1]:56038: use of closed network connection
  Stacktrace:
```

@haydonduan Do you also see the similar stack trace on the console along with the error message you mentioned?
